src/apps/lwaftr/acl.lua

```diff
@@ -0,0 +1,87 @@
+module(...,package.seeall)
+
+local filter = require("lib.pcap.filter")
+
+local function trim(s)
+  return s:gsub("^%s*(.-)%s*$", "%1")
+end
+
+local function columns(content, sep)
+   local function iter(state)
+      local token, pos = state.content:match(state.regex, state.start)
+      if not pos then return nil end
+      state.start = pos
+      return token
+   end
+   sep = sep or ","
+   local regex = ("([^%s]+)()%s"):format(sep, sep)
+   local state = { content = content, regex = regex, start = 0 }
+   return iter, state
+end
+
+--[[
+'filename' is an ACL file. It's encoded as a CSV with the following structure:
+
+   ipv4_ingress_filter,ipv6_ingress_filter,ipv4_egress_filter,ipv6_egress_filter
+   rule11,rule21,,rule41
+   rule12,rules22,,
+   rule13,,,
```

Comment:
It could indeed be that pflang is the right way to go, for what it's worth.  If that's the case, then we could simply replace any thought of CSV files with conf parameters directly, i.e. ipv4_ingress_filter = ...  Let's talk about it tomorrow.